src/init.cpp

```diff
@@ -188,7 +188,7 @@ void Shutdown()
     if (!lockShutdown)
         return;
 
-    /// Note: Shutdown() must be able to handle cases in which AppInit2() failed part of the way,
+    /// Note: Shutdown() must be able to handle cases in which AppInitMain() failed part of the way,
```

Comment:
For documentation/string commits like this, amending is best IMO.
For more involved commits, it can make sense to keep a history, at least until merge.